apps/src/tests/index.ts

```diff
@@ -141,6 +141,7 @@ export { default as Test2899 } from './Test2899';
 export { default as Test2926 } from './Test2926'; // [E2E created](iOS): PR related to iOS search bar
 export { default as Test2949 } from './Test2949'; // [E2E skipped]: can't check system bars styles
 export { default as Test2963 } from './Test2963'; // [E2E created](iOS): issue related to iOS
+export { default as TestPR3045 } from './TestPR3045';
```

Comment:
```suggestion
export { default as Test3045 } from './Test3045';
```
To follow the convention, I'd just use the number here + in filename.